src/Model/Completions.re

```diff
@@ -21,101 +21,98 @@ let default: t = {
   meet: None,
   selected: None,
   filter: None,
-  filteredCompletions: [{
-    completionLabel: "log",
-    completionKind: CompletionKind.Method,
-    completionDetail: Some("() => ()"),
-  }, {
-    completionLabel: "warn",
-    completionKind: CompletionKind.Method,
-    completionDetail: None,
-  }, {
-    completionLabel: "error",
-    completionKind: CompletionKind.Method,
-    completionDetail: None,
-  }],
-  completions: [{
-    completionLabel: "log",
-    completionKind: CompletionKind.Method,
-    completionDetail: Some("() => ()"),
-  }, {
-    completionLabel: "warn",
-    completionKind: CompletionKind.Method,
-    completionDetail: None,
-  }, {
-    completionLabel: "error",
-    completionKind: CompletionKind.Method,
-    completionDetail: None,
-  }],
-}
+  filteredCompletions: [],
+  completions: [],
+};
 
 let isActive = (v: t) => {
   switch (v.meet) {
   | None => false
-  | Some(_) => switch(v.filteredCompletions) {
-  | [_hd, ..._tail] => true
-  | _ => false
-  }
-  }
+  | Some(_) =>
+    switch (v.filteredCompletions) {
+    | [_hd, ..._tail] => true
+    | _ => false
+    }
+  };
```

Comment:
I think this can be reduced to `v.meet != None && v.filteredCompletions != []`

Reply:
Good call!